### PR TITLE
update docs to include requirement of redirect url for visa OB; note …

### PR DESCRIPTION
**Pull request by spencerhunter**
…change on changelog

### CODE REVIEW

File: pages/changelog.mdx [hunk 1]
```diff
@@ -12,6 +12,21 @@ Keep up to date on new product features and enhancements, API docs, tools change
...
+**CHANGED**
+
+##### Description:
+
+- **Multiple Redirect URLs:** You can now configure multiple redirect URLs for your exchange sessions, providing more flexibility in your integration.
+- **Required Redirect URL:** A redirect URL is now required when [creating an exchange session](https://developers.dwolla.com/docs/balance/api-reference/open-banking/create-exchange-session-for-a-customer) to ensure proper validation and security.
+
+##### Changes:
+
+- The [create an exchange session API endpoint](https://developers.dwolla.com/docs/balance/api-reference/open-banking/create-exchange-session-for-a-customer) now requires a redirect url parameter, which is a single URL that is already configured with Dwolla.
```
Comment (spencerhunter) [inline pages/changelog.mdx]: Recommendations on /balance and /connect links ? 🤔

File: pages/changelog.mdx [hunk 2]
```diff
@@ -12,6 +12,21 @@ Keep up to date on new product features and enhancements, API docs, tools change
 
 ## Upcoming
 
+### 2024-10-30
+
+**CHANGED**
+
+##### Description:
+
+- **Multiple Redirect URLs:** You can now configure multiple redirect URLs for your exchange sessions, providing more flexibility in your integration.
```
Comment (ShreyaThapa) [inline pages/changelog.mdx:21]: Maybe I'm mistaken on the timeline, should this be in the **Upcoming** section?

Comment (ShreyaThapa) [reply pages/changelog.mdx:21]: NVM! Didn't check properly 😅

File: pages/changelog.mdx (hunk 1 above)
Comment (ShreyaThapa) [reply pages/changelog.mdx]: Maybe something like this?

```suggestion
- The endpoint for creating an exchange session now requires a redirect url parameter, which is a single URL that is already configured with Dwolla ([Balance docs](https://developers.dwolla.com/docs/balance/api-reference/open-banking/create-exchange-session-for-a-customer), [Connect docs](https://developers.dwolla.com/docs/connect/api-reference/open-banking/create-exchange-session-for-an-external-party)).
```

File: pages/changelog.mdx [hunk 3]
```diff
@@ -12,6 +12,21 @@ Keep up to date on new product features and enhancements, API docs, tools change
...
+- **Required Redirect URL:** A redirect URL is now required when [creating an exchange session](https://developers.dwolla.com/docs/balance/api-reference/open-banking/create-exchange-session-for-a-customer) to ensure proper validation and security.
+
+##### Changes:
+
+- The [create an exchange session API endpoint](https://developers.dwolla.com/docs/balance/api-reference/open-banking/create-exchange-session-for-a-customer) now requires a redirect url parameter, which is a single URL that is already configured with Dwolla.
+- The validation process for redirect URLs has been enhanced to ensure they meet security standards and are accessible. The exchange sessions API endpoint will return an HTTP 400 `ValidationError` with an `Invalid` code and message of `The provided redirect URL must exactly match one of the configured URLs for the account`.
+- Refer to the [developer documentation](https://developers.dwolla.com/docs/balance/open-banking/visa) for detailed usage instructions and [API reference](https://developers.dwolla.com/docs/balance/api-reference/open-banking/create-exchange-session-for-a-customer).
```
Comment (ShreyaThapa) [inline pages/changelog.mdx]: Following the above example for linking to both Balance and Connect docs.
```suggestion
- Refer to the [developer documentation](https://developers.dwolla.com/docs/balance/open-banking/visa) for detailed usage instructions and API reference ([Balance](https://developers.dwolla.com/docs/balance/api-reference/open-banking/create-exchange-session-for-a-customer), [Connect](https://developers.dwolla.com/docs/connect/api-reference/open-banking/create-exchange-session-for-an-external-party)).
```

File: pages/changelog.mdx (hunk 1 above)
Comment (spencerhunter) [reply pages/changelog.mdx]: Yeah, that's what I was thinking

File: pages/docs/balance/api-reference/open-banking/create-exchange-session-for-a-customer.mdx [hunk 4]
```diff
@@ -11,30 +11,36 @@ meta:
 
 # Create an exchange session for a customer
 
-This section details how to initiate Instant Account Verification for a customer using Dwolla's trusted open banking partners. You can trigger this process by creating an exchange session. There are two supported partners currently available: Visa and MX.
+To initiate Instant Account Verification for a customer using Dwolla's trusted open banking partners, you'll need to create an exchange session. Currently, Visa and MX are supported partner.
```
Comment (ShreyaThapa) [inline pages/docs/balance/api-reference/open-banking/create-exchange-session-for-a-customer.mdx]:
```suggestion
To initiate Instant Account Verification for a customer using Dwolla's trusted open banking partners, you'll need to create an exchange session. Currently, Visa and MX are the supported partners.
```

File: pages/docs/balance/api-reference/open-banking/create-exchange-session-for-a-customer.mdx [hunk 5]
```diff
@@ -11,30 +11,36 @@ meta:
...
+| -------- | -------- | ------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| \_links  | yes      | object | A HAL-JSON link that represents the Exchange Partner resource. This identifies the open banking provider used for creating the exchange session. **Note:** When using Visa as the exchange partner, a redirect url must be provided within the \_links object. This redirect URL will be validated against a previously configured redirect URL stored in Dwolla. The validated redirect URL will be used to redirect the user back to your application after they complete the authorization process with Visa. |
```
Comment (ShreyaThapa) [inline pages/docs/balance/api-reference/open-banking/create-exchange-session-for-a-customer.mdx]:
```suggestion
| \_links  | yes      | object | A HAL-JSON link that represents the Exchange Partner resource. This identifies the open banking provider used for creating the exchange session. **Note:** When using Visa as the exchange partner, a redirect url must also be provided within the \_links object. This redirect URL will be validated against a previously configured redirect URL stored in Dwolla. The validated redirect URL will be used to redirect the user back to your application after they complete the authorization process with Visa. |
```

File: pages/docs/balance/api-reference/open-banking/create-exchange-session-for-a-customer.mdx [hunk 6]
```diff
@@ -11,30 +11,36 @@ meta:
...
+| 201         | -            | -                                                                                        | The Dwolla API accepted the request and created an exchange session resource. You can reference the Location header to retrieve a link that represents the created exchange session resource. |
+| 400         | Invalid      | /\_links/exchange-partner/href is invalid.                                               | ValidationError. Returned if the exchange partner URL is invalid.                                                                                                                             |
+| 400         | Invalid      | /\_links/redirect-url/href                                                               | ValidationError. Returned if the redirect URL is invalid.                                                                                                                                     |
```
Comment (ShreyaThapa) [inline pages/docs/balance/api-reference/open-banking/create-exchange-session-for-a-customer.mdx]:
```suggestion
| 400         | Invalid      | /\_links/redirect-url/href  is invalid.                                                             | ValidationError. Returned if the redirect URL is invalid.                                                                                                                                     |
```

File: pages/docs/connect/api-reference/open-banking/create-exchange-session-for-an-external-party.mdx [hunk 7]
```diff
@@ -11,28 +11,34 @@ meta:
 
 # Create an exchange session for an external party
 
-This section details how to initiate Instant Account Verification for an <span><ContentTooltip preset="external-party">external party</ContentTooltip></span> using Dwolla's trusted open banking partners. You can trigger this process by creating an exchange session. There are two supported partners currently available: Visa and MX.
+To initiate Instant Account Verification for an external party using Dwolla's trusted open banking partners, you'll need to create an exchange session. Currently, Visa and MX are supported partner.
```
Comment (ShreyaThapa) [inline pages/docs/connect/api-reference/open-banking/create-exchange-session-for-an-external-party.mdx]:
```suggestion
To initiate Instant Account Verification for an external party using Dwolla's trusted open banking partners, you'll need to create an exchange session. Currently, Visa and MX are the supported partners.
```

File: pages/docs/connect/api-reference/open-banking/create-exchange-session-for-an-external-party.mdx [hunk 8]
```diff
@@ -11,28 +11,34 @@ meta:
...
+| 201         | -            | -                                                                                        | The Dwolla API accepted the request and created an exchange session resource. You can reference the Location header to retrieve a link that represents the created exchange session resource. |
+| 400         | Invalid      | /\_links/exchange-partner/href is invalid.                                               | ValidationError. Returned if the exchange partner URL is invalid.                                                                                                                             |
+| 400         | Invalid      | /\_links/redirect-url/href                                                               | ValidationError. Returned if the redirect URL is invalid.                                                                                                                                     |
```
Comment (ShreyaThapa) [inline pages/docs/connect/api-reference/open-banking/create-exchange-session-for-an-external-party.mdx]:
```suggestion
| 400         | Invalid      | /\_links/redirect-url/href is invalid.                                                               | ValidationError. Returned if the redirect URL is invalid.                                                                                                                                     |
```

**Review by ShreyaThapa — COMMENTED** [review]
My comments an suggestions are all in!

File: pages/docs/connect/api-reference/open-banking/create-exchange-session-for-an-external-party.mdx (hunk 7 above)
Comment (ShreyaThapa) [reply pages/docs/connect/api-reference/open-banking/create-exchange-session-for-an-external-party.mdx]: Thanks for applying the suggestions, @spencerhunter! 🙌 

There's just this one last wee little 🤏 suggestion. 🙂

**Review by ShreyaThapa — APPROVED** [review]
🎉

**Pull request merged**